lldb/test/API/functionalities/target-new-solib-notifications/Makefile

```diff
@@ -1,3 +1,23 @@
-CXX_SOURCES := main.cpp
-
-include Makefile.rules
+CXX_SOURCES := main.cpp
+LD_EXTRAS := -L. -lloadunload_d -lloadunload_c -lloadunload_a -lloadunload_b

```

Comment:
Please rename the modules to something else (even a simple `liba` is probably fine).

Reply:
Done.